server/server.go

```diff
@@ -53,7 +55,9 @@ type Server struct {
 	startedTs int64
 	secret    string
 
-	// boot specifies that the server
+	mysqlbinlogDir string
```

Comment:
Instead of making the mysqlbinlog path variable, why not just install it to a constant place?

Reply:
IMO, mysqlbinlog binary should be install under the `resources` directory, but now, `resources` directory is determined by `metadataDB`. You can check here: https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/store/metadb.go?L41

And if we install it to a constant place and not save it in a var to record it, then we need to generate this path by ourselves near the download binlog, and I don't think this is a good choice. We can try to keep the tree structure.

Reply:
`resources` is where we store static blobs.
When Bytebase server is running, we should install things into the `Profile.DataDir` like this
https://github.com/bytebase/bytebase/blob/504d9fef872be2935e4de293ba634a469ce81879/store/metadb.go#L42

Reply:
I prefer to extract the logic of the `resources` path, the mysqlbinlog binary should be the same level as the postgres binary.

Reply:
SG.
Maybe we can still remove this field and compute the `mysqlbinlog` bin path in a function.